webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/EndpointsDefinition.scala

```diff
@@ -167,6 +167,20 @@ object ComplianceApi       extends ApiModuleProvider[ComplianceApi] {
     val dataContainer  = Some("directiveCompliance")
   }
 
+  final case object GetDirectivesCompliance extends ComplianceApi with ZeroParam with StartsAtVersion10 with SortIndex {
+    val z              = implicitly[Line].value
+    val description    = "Get all directive's compliance"
+    val (action, path) = GET / "compliance" / "directives"
+    val dataContainer  = Some("directivesCompliance")
+  }
+
+  final case object ExportDirectiveComplianceCSV extends ComplianceApi with OneParam with StartsAtVersion10 with SortIndex {
```

Comment:
I would rather have the URL to be compliance/directives/id/csv

Reply:
(in fact i would prefer compliance/directives/id?format=csv

Reply:
Yes i would définitely prefer one endpoint and choose the format as a parameter

Reply:
why does it starts at version 10 and not the latest version ?